src/app/shared/resource/header/resource-header.component.html

```diff
@@ -7,10 +7,10 @@
      [style.color]="resource.color" data-toggle="dropdown" (click)="storeIconAndColor($event)"></i>
   <div class="dropdown-menu">
     <color-picker dropdownId="dropdown-header" class="dropdown-item" [(color)]="resource.color"
-                  ></color-picker>
+    ></color-picker>
```

Comment:
I have seen this kind of formatting quite often recently (not only at this project) so we may need to start using [Prettier](https://prettier.io/) soon in order to focus on business logic and avoid such formatting issues.